pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -24,6 +24,7 @@
 #include "LinearAlgebra.hpp"
 #include "PauliGenerator.hpp"
 
+#include <bit>
```

Comment:
I think we don't need to include this here or even in `GateImplementationsPI.hpp` as `BitUtil.hpp` is already included. right? 
```suggestion
```

Reply:
Better to be explicit about any needed headers. This can cause havoc if we ever refactor and the header goes away in the other file. Best to keep this in wherever used.

Reply:
Excellent point! Thanks @mlxd